regulations/generator/layers/footnotes.py

```diff
@@ -0,0 +1,22 @@
+from regulations.generator.layers.base import LayerBase
+
+
+class FootnotesLayer(LayerBase):
+    """Assembles the footnotes for this node, if available"""
+    shorthand = 'footnotes'
+    data_source = 'formatting'
+    layer_type = LayerBase.PARAGRAPH
+
+    def __init__(self, layer, version=None):
+        self.layer = layer
+        self.version = version
+
+    def apply_layer(self, text_index):
+        """Return a tuple of 'footnotes' and collection of footnotes"""
+        footnotes = []
+        for label in self.layer.keys():
+            if label.startswith(text_index):
```

Comment:
I understand, but I'm betting the "why is this using `startswith` rather than equality" will throw us in the future. Would you mind commenting about this either here of in the function comments?